AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -273,6 +273,8 @@ consent-pref.trustarc.com#%#//scriptlet('trusted-click-element', 'div[aria-label
 !
 ! NOTE: Popular rules end ⬆️
 ! !SECTION: Cookies - Popular rules
+the-decoder.com##iframe#sp_message_iframe_929061
+the-decoder.com##div#sp_message_container_929061
```

Comment:
Wrong place.
Why not to block the cookie management system script?

Reply:
Use search in resources to find the necessary script.

Reply:
should I change the place or change the rule?

Reply:
Both. Find the script which loads CMP.